src/components/AuditLogsTable/AuditLogsTablePagination/AuditLogsTablePagination.tsx

```diff
@@ -93,8 +102,8 @@ const AuditLogsTablePagination = ({
           scrollToTop();
         }}
         // No further pages if couldn't fetch (limit + 1) entries.
-        forwardDisabled={(auditLogs?.length ?? 0) <= limit}
-        backDisabled={page === 1}
+        forwardDisabled={auditLogsLoading || !auditLogsLoaded || !hasNextPage}
```

Comment:
It might be good to add tests for disabling the button when loading.

Reply:
Good idea! I've added the tests.